src/subscript/sector2fluxnum/sector2fluxnum.py

```diff
@@ -0,0 +1,251 @@
+from ecl.eclfile import EclFile, FortIO
+from subscript.sector2fluxnum import flux_obj
+from subscript.sector2fluxnum import fluxfile_obj as ffo
+from subscript.sector2fluxnum import datafile_obj as do
+from subscript.sector2fluxnum import completions as comp
+
+
+DESCRIPTION = """
+This is a wrapper script that executes the different
+steps in the workflow for generating sector models in ECLIPSE.
+
+The sector models have the same resolution as the full-field model
+
+Further details about usage and howto can be found here:
+
+http://wiki.equinor.no/wiki/index.php/Res:Gen_FLUX_fipnum_region.py
+
+Author: Thomas Ramstad, trams@equinor.com .
+
+"""
+
+
+def header_print():
+    """ """
+    sys.stdout.write(
+        "*********************************************************************\n"
+        "*********************************************************************\n"
+        "** This is a wrapper script for making sector models               **\n"
+        "** with full field resulution.                                     **\n"
+        "**                                                                 **\n"
+        "** Never forget to QC results of a script: The user is responsible **\n"
+        "** for making sure the output of a script is correct.              **\n"
+        "**                                                                 **\n"
+        "*********************************************************************\n"
+        "*********************************************************************\n"
+    )
+
+    print("")
+
+
+def get_parser():
+    """Function to create the argument parser that is going to be served to the user.
+
+    Returns:
+    parser (argparse.ArgumentParser): The argument parser to be served
+
+    """
+
+    parser = argparse.ArgumentParser(prog="sector2fluxnum.py", description=DESCRIPTION)
+    parser.add_argument("ECLIPSE_CASE", type=str, help="Eclipse input case")
+    parser.add_argument("OUTPUT_CASE", type=str, help="Name of output FLUX file")
+    parser.add_argument("-i", type=str, help="Example: -i 5-20")
+    parser.add_argument("-j", type=str, help="Example: -j 5-20")
+    parser.add_argument("-k", type=str, help="Example: -k 5-20")
+    parser.add_argument("-f", "--fipnum", type=str, help="Example: -f 46")
+    parser.add_argument(
+        "-r", "--restart", type=str, help="Restart file read separately"
+    )
+    parser.add_argument("-e", "--egrid", type=str, help="EGRID file read separately")
+    parser.add_argument(
+        "-w", "--well", action="store_true", help="Well data read from RST file"
+    )
+    parser.add_argument("--fipfile", type=str, help="FIPNUM file read separately")
+    parser.add_argument("--fluxfile", type=str, help="FLUXNUM file read separately")
+    parser.add_argument(
+        "--test", type=str, help="Name of predefined DUMPFLUX case for testing"
+    )
+    parser.add_argument("-v", "--version", type=str, help="ECL version")
```

Comment:
This options should be renamed into "--ecl-version" or something. "--version" as a command line option  is most often used to print out the version of the tool itself, not as an option supplied to third party code.

Reply:
See e.g. https://github.com/equinor/subscript/issues/367

Reply:
renamed to "ecl_version".